catalog-rest-service/src/main/java/org/openmetadata/catalog/resources/mlmodels/MlModelResource.java

```diff
@@ -103,7 +103,7 @@ public MlModelList(List<MlModel> data, String beforeCursor, String afterCursor,
   }
 
   static final String FIELDS =
-      "owner,dashboard,algorithm,mlFeatures,mlHyperParameters,mlStore,server," + "followers,tags,usageSummary";
+      "owner,dashboard,algorithm,mlFeatures,mlHyperParameters,mlStore,server,target," + "followers,tags,usageSummary";
```

Comment:
Remove the + concatenation, since it is in one line?

Reply:
Good point, missed the autoformat here. On it